video/video.ino

```diff
@@ -65,16 +71,15 @@
 bool					terminalMode = false;	// Terminal mode
 fabgl::Terminal			Terminal;				// Used for CP/M mode
 VDUStreamProcessor *	processor;				// VDU Stream Processor
-
-#if DEBUG == 1 || SERIALKB == 1
-HardwareSerial DBGSerial(0);
-#endif 
+#if ZDI==1
+	#include "zdi.h"
+#endif
 
 void setup() {
 	disableCore0WDT(); delay(200);				// Disable the watchdog timers
 	disableCore1WDT(); delay(200);
-	#if DEBUG == 1 || SERIALKB == 1
-	DBGSerial.begin(500000, SERIAL_8N1, 3, 1);
+	#if DEBUG == 1 || SERIALKB == 1 || ZDI == 1
+	DBGSerial.begin(460800, SERIAL_8N1, 3, 1); // not stable at 500000

```

Comment:
heh - curious.  that speed seems to be stable for me.

there's similar issues of stability around the Arduino upload speed where some people report on M1 Macs not being able to use higher speeds, but I'm running 921600 upload speed on my M1Max, and DBGSerial at 500000...

maybe we should just `#define` this speed above so it's configurable

Reply:
My M1Pro with a USB3 to USB-C cable to the Olimex Agon can only do max 460800. Similarly when uploading MOS from Arduino I have to lower it to 460800. Maybe I should change cables to rule out any issues with that. 
Your suggestion to make a define allows us to change it at one logical place. I'll update.

Reply:
heh - yeah - I did initially see issues trying to upload firmware updates at high speed on my Mac when I was primarily working with my Olimex Agon Light 2 as well, and had to drop down to a lower speed...

I don't remember changing to the higher speed, but the Arduino IDE is mysterious in such matters. :grin:  the Console8 seems completely happy at the higher speeds, and I don't seem to have to slow it down when I plug the Olimex in.  All very strange!